api/tests/src/db/models/factories.py

```diff
@@ -1907,4 +1913,16 @@ class Meta:
     opportunity = factory.SubFactory(OpportunityFactory)
     opportunity_id = factory.LazyAttribute(lambda o: o.opportunity.opportunity_id)
 
-    opportunity_data = factory.LazyAttribute(lambda o: o.opportunity_data)
+    opportunity_data = factory.LazyAttribute(lambda o: SCHEMA.dump(o.opportunity))
+
+
+class JobLogFactory(BaseFactory):
+    class Meta:
+        model = task_models.JobLog
+
+    job_id = Generators.UuidObj
+    job_type = factory.LazyAttribute(
```

Comment:
This is the sort of field that if you want to use a factory for it, you should pass the value in yourself since a random value is going to be weird with tests (since you usually care a lot about the exact value).